samples/nextjs/scripts/generate-config.ts

```diff
@@ -33,6 +33,7 @@ export function generateConfig(configOverrides?: { [key: string]: string }): voi
   // based on other config settings.
   const computedConfig: { [key: string]: string } = {};
   computedConfig.graphQLEndpoint = '`${config.sitecoreApiHost}${config.graphQLEndpointPath}`';
+  computedConfig.sitecoreServicesApiKey = 'config.sitecoreApiKey';
```

Comment:
In case the Experience Edge is used for content delivery and Sicore CD for personalization decision making two keys are required. This is approved by Product Management.